### PR TITLE
Upgraded to Swagger UI 3.4.2

### PR DESCRIPTION
**Pull request by dmolesUC**
Fix for #108.

### CODE REVIEW

File: src/main/resources/swagger-ui/index.html [hunk 1]
```diff
@@ -71,12 +71,12 @@
 <script src="./swagger-ui-standalone-preset.js"> </script>
 <script>
 window.onload = function() {
-
+  
   // Build a system
   const ui = SwaggerUIBundle({
-    spec: /*intellij-swagger-spec-start*//*intellij-swagger-spec-end*/,
```
Comment (semonte) [inline src/main/resources/swagger-ui/index.html:77]: We need the `spec` and `validatorUrl`, the spec is where the plugin will inject the JSON Open API specification.

File: src/main/resources/swagger-ui/index.html [hunk 2]
```diff
@@ -71,12 +71,12 @@
...
-    validatorUrl: null,
+    url: "http://petstore.swagger.io/v2/swagger.json",
```
Comment (semonte) [inline src/main/resources/swagger-ui/index.html:77]: This should be removed, we don't want to render the petstore specification.

**Comment by dmolesUC** [comment]
OK. I don't have time to work on this at the moment and in the interim 3.4.3 has come out, so if it's not a trivial drop-in, you should probably reject this PR in favor of a fix from someone who actually knows the plugin code.

**Comment by semonte** [comment]
Ok, thanks for the effort anyway!

**Pull request closed without merging**